prow/plugins/lgtm/lgtm.go

```diff
@@ -30,9 +30,13 @@ import (
 	"k8s.io/test-infra/prow/repoowners"
 )
 
+// PluginName needs to be exported for checkconfig
 const PluginName = "lgtm"
 
 var (
+	addLGTMLabelNoti   = "LGTM label has been added.  <details>%s</details>"
```

Comment:
What is `Noti`?

Reply:
Notification... I saw the same in approve plugin if I remember correctly.
Of course it can be changed!

Reply:
Let's just use the word `Notification` -- I was trying to figure out what `"not i"` meant :)